geckodriver.log

```diff
@@ -0,0 +1,1895 @@
+1622233777318	geckodriver	INFO	Listening on 127.0.0.1:53699
```

Comment:
You should add this file to the `.gitignore`

Also, since it's already being tracked, you'll need to either explicitly remove it from the repo (`git rm --cached geckodriver.log`) or just delete it